### PR TITLE
Use alpine and env for AWS keys.

### PR DESCRIPTION
**Pull request by ptondereau**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,17 +2,22 @@
...
 ## Install
-Access your API keys [here](https://www.scaleway.com/en/docs/generate-an-api-token/)
+Get your API keys [here](https://www.scaleway.com/en/docs/generate-an-api-token/)
 
-### Test with docker
-Just type: `docker-compose up`
+### Usage
+```bash
+$ docker build -t scaleways3 .
+$ docker run scaleways3 -e AWS_ACCESS_KEY_ID=XXXXX -e AWS_SECRET_ACCESS_KEY=XXXX
```
Comment (mrpandat) [inline README.md]: `$ docker run --env AWS_ACCESS_KEY_ID=XXXXX --env AWS_SECRET_ACCESS_KEY=XXXX scaleways3`

**Comment by mrpandat** [comment]
Thanks for the PR!
But, the login system via Scaleway does not work with the environment variables. You have to generate your `.aws/credentials` file.

A solution fort this problem ?

File: src/index.php [hunk 2]
```diff
@@ -19,18 +19,20 @@
 
 require 'vendor/autoload.php';
 
+use Aws\S3\Exception\S3Exception;
 use Aws\S3\S3Client;
-use Aws\Exception\AwsException;
 
 //Create a S3Client 
 $s3 = new S3Client([
-    'profile' => 'default',
```
Comment (ptondereau) [inline src/index.php:27]: https://github.com/aws/aws-sdk-php/blob/e4672392e63e91085cde5f1fbcbf5883a69b26a0/src/AwsClient.php#L142

Comment (mrpandat) [reply src/index.php:27]: image

**Comment by mrpandat** [comment]
GG <3

**Pull request merged**